course/enrollment.py

```diff
@@ -178,6 +179,15 @@ def decide_enrollment(approved, modeladmin, request, queryset):
             participation.status = participation_status.denied
         participation.save()
 
+        send_enrollment_decision(participation, approved, request)
+
+        count += 1
+
+    messages.add_message(request, messages.INFO,
+            # Translators: how many enroll requests have ben processed.
+            _("%d requests processed.") % count)
+
+def send_enrollment_decision(participation, approved, request):
         with translation.override(settings.RELATE_ADMIN_EMAIL_LOCALE):
```

Comment:
According to https://github.com/inducer/relate/issues/90, seems it's not a problem to use which locale when sending email to student, I think it is a feature that a student receive notifications using his/her prefered language.